clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -169,7 +169,9 @@ export const CustomMultiSelect = ({
 }: SelectProps & FieldHookConfig<string[]>) => {
   const [field, meta] = useField(props);
   const isInvalid = !!(meta.touched && meta.error);
-  const selected = options.filter((o) => field.value.indexOf(o.value) >= 0);
+  const selected = field.value
+    ? options.filter((o) => field.value.indexOf(o.value) >= 0)
+    : [];
```

Comment:
Was this a problem because the value is coming through as `null`/`undefined` when the "blank" state should be an empty string?